src/tools/dart-devtools.md

```diff
@@ -8,6 +8,8 @@ for Dart and Flutter.
 These tools are distributed as part of the `dart` tool
 and interact with tools such as IDEs, `dart run`, and `webdev`.
 
+<img src="/tools/DevTools.png" width="400">
```

Comment:
Seems a bit small for me. Also can we add some sort of alt text?

```suggestion
<img src="/tools/DevTools.png" width="640" alt="Screenshot of DevTools' Memory page">
```

Reply:
Agreed, very small